scripts/release/mule/package/linux/amd64/docker/package.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo
+date "+build_release begin PACKAGE DOCKER stage %Y%m%d_%H%M%S"
+echo
+
+OS_TYPE="$1"
+ARCH="$2"
+WORKDIR="$3"
+
+if [ -z "$OS_TYPE" ] || [ -z "$ARCH" ] || [ -z "$WORKDIR" ]; then
+    echo OS, ARCH and WORKDIR variables must be defined.
```

Comment:
don't you need to place these in double-quotes ?

Reply:
Changed.